src/components/__tests__/LibraryListTable.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render } from '@test-utils';
+import { test } from 'vitest';
+import { composeStories } from '@storybook/testing-react';
+import * as stories from '../__stories__/LibraryListTable.stories';
+import { ChakraProvider } from '@chakra-ui/react';
+
+const { Default: LibraryListTable } = composeStories(stories);
+
+test('renders without crashing', () => {
+  render(
+    <ChakraProvider>
```

Comment:
this provider could be added to test-utils https://github.com/adsabs/nectar/blob/56d14cdfed266797b90d3e737fa5474cdeb2c506/src/test-utils.tsx#L63